### DISCUSSION TITLE
3D semantic segmentation loss curve oscillates

### DISCUSSION THREAD

**Discussion by lanslotttTT** [Project-MONAI/MONAI#4113] [Q&A]
When I implement 3D segmentation with monai, I find that my loss curve oscillates ![2022-04-12 23-04-05 的屏幕截图](https://user-images.githubusercontent.com/30655387/162993359-56df3ffc-0989-474f-b4e6-9a77f0a5ac3c.png), is this a good or bad thing?
The data on my validation set doesn't look different and dice is able to calculate correctly
![2022-04-12 23-06-46 的屏幕截图](https://user-images.githubusercontent.com/30655387/162993942-6a00dff1-2973-493d-90ce-5a1a9fc25b91.png)

**Comment by Nic-Ma** [comment]
Hi @lanslotttTT ,

Thanks for your interest here.
I think maybe you can try a bigger smooth factor for the loss curve?

Thanks.
